packages/hoprd/Dockerfile

```diff
@@ -32,7 +32,10 @@ RUN yarn buildAdmin
 
 # -- RUNTIME STAGE ------------------------------\
 
-FROM node:12.9.1-buster AS runtime
+FROM node@sha256:966f569423c977e862b4e5b1f01234720dd212394cd13f5904d47864aece5417 AS runtime
+
+RUN wget https://github.com/Yelp/dumb-init/releases/download/v1.2.4/dumb-init_1.2.4_amd64.deb
```

Comment:
What does this do?

Reply:
Fetches version `1.2.4` of [dumb-init](https://github.com/Yelp/dumb-init). We are using it as a replacement of a PID 1 for docker. From the [linked article](https://snyk.io/blog/10-best-practices-to-containerize-nodejs-web-applications-with-docker/): 

```
When processes run as PID 1 they effectively take on some of the responsibilities of an init system, which is typically responsible for initializing an operating system and processes. The kernel treats PID 1 in a different way than it treats other process identifiers. This special treatment from the kernel means that the handling of a SIGTERM signal to a running process won’t invoke a default fallback behavior of killing the process if the process doesn’t already set a handler for it.

To quote the Node.js Docker working group recommendation on this:  “Node.js was not designed to run as PID 1 which leads to unexpected behaviour when running inside of Docker. For example, a Node.js process running as PID 1 will not respond to SIGINT (CTRL-C) and similar signals”.

The way to go about it then is to use a tool that will act like an init process, in that it is invoked with PID 1, then spawns our Node.js application as another process whilst ensuring that all signals are proxied to that Node.js process. If possible, we’d like a small as possible tooling footprint for doing so to not risk having security vulnerabilities added to our container image.

One such tool that we use at Snyk is dumb-init because it is statically linked and has a small footprint.
```

I've used [tini](https://github.com/krallin/tini) in the past, and was aware of this pitfall but never paid too much attention to it. Sadly this is probably the main reason why our container dies and does not get picked up again by [konlet](https://github.com/GoogleCloudPlatform/konlet), which is the orchestration engine to start and stop containers in our `gcloud` images.

Reply:
can you put a short comment?